arch/arm64/configs/enchilada_defconfig

```diff
@@ -742,18 +742,67 @@ CONFIG_AUTOFS4_FS=y
 
 CONFIG_LOCALVERSION="-hybris"
 
-# Recommended
+# Memory
 CONFIG_MEMCG=y
 CONFIG_MEMCG_KMEM=y
 CONFIG_MEMCG_SWAP=y
 
-
-# telnet
-
+# Telnet
 CONFIG_USB_F_RNDIS=y
 CONFIG_USB_CONFIGFS_RNDIS=y
 CONFIG_USB_F_GSI=n
 CONFIG_USB_CONFIGFS_F_GSI=n
 CONFIG_USB_CONFIGFS_F_QCGSI=n
 CONFIG_USB_CONFIGFS_QCRNDIS=n
-CONFIG_USB_CONFIGFS_RMNET_BAM=n 
\ No newline at end of file
+CONFIG_USB_CONFIGFS_RMNET_BAM=n
+
+# EXT4
+CONFIG_LBDAF=y # suport large ext4
+
+# CIFS
+CONFIG_CIFS=y
+CONFIG_BLK_DEV_NBD=y
+
+# NFS File System
+CONFIG_LOCKD=y
+CONFIG_LOCKD_V4=y
+CONFIG_NFS_V3_ACL=y
+CONFIG_NFS_FS=y
+CONFIG_NFS_COMMON=y
+CONFIG_NFS_V4=y
+CONFIG_NFS_V4_1=y
+CONFIG_NFS_V3=y
+CONFIG_SUNRPC_GSS=y
+CONFIG_NFS_USE_KERNEL_DNS=y
+CONFIG_SUNRPC=y
+CONFIG_NFS_ACL_SUPPORT=y
+
+# CD File Ssytem / Portable USB (OTG)
+CONFIG_UDF_FS=y
+CONFIG_ISO9660_FS=y
+
+# BTRFS
+#CONFIG_BTRFS_FS=y # SailFish's default FS
+
+# Networking
+CONFIG_NETFILTER_XT_MATCH_SCTP=y # for NFS support
+CONFIG_NETFILTER_XT_MATCH_RECENT=y # IPHTaxren recent match
+CONFIG_IP6_NF_MATCH_AH=y # IPv6 recent match
+CONFIG_IP6_NF_MATCH_MH=y # mh match
+CONFIG_NF_NAT_IPV6=y # IPv6 NAT
+CONFIG_IP6_NF_MATCH_FRAG=y # IPV6 Frag match
+CONFIG_NETFILTER_XT_MATCH_DCCP=y # IPTables DCCP match
+
+# Clock
+CONFIG_RTC_DRV_CMOS=y
```

Comment:
```
Say "yes" here to get direct support for the real time clock found in every PC or ACPI-based system, and some other boards. Specifically the original MC146818, compatibles like those in PC south bridges, the DS12887 or M48T86, some multifunction or LPC bus chips, and so on.
```
[from here](https://cateee.net/lkddb/web-lkddb/RTC_DRV_CMOS.html)

RTC works fine on this device... We don't need this option